File: PracticeAndroid/AndroidPractice2/MyFolder/LOCOMO/app/src/main/java/com/example/locomo/fragment/ThirdFragment.java
package com.example.locomo.fragment;

import android.os.Bundle;

import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.example.locomo.R;
import com.example.locomo.activity.DashBoardActivity;

public class ThirdFragment extends Fragment {
    Button closeOutDay_btn;
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View view= inflater.inflate(R.layout.fragment_third, container, false);
//        ((DashBoardActivity) getActivity()).stopTimer();
        closeOutDay_btn=view.findViewById(R.id.closeOutDay_btn);
        closeOutDay_btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
               boolean val= ((DashBoardActivity) getActivity()).isLocationServiceRunning();
               if (val){
                   ((DashBoardActivity) getActivity()).stopTimer();
               }
            }
        });
        return view;
    }
}

File: PracticeAndroid/Andorid_Kotlin/ADExpress/Camerax_MVVM/app/src/main/java/com/androdevdk/camerax_mvvm/test/fragments/ThirdFragment.kt
package com.androdevdk.camerax_mvvm.test.fragments

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.androdevdk.camerax_mvvm.R
import com.androdevdk.camerax_mvvm.databinding.FragmentCameraBinding
import com.androdevdk.camerax_mvvm.databinding.FragmentThirdBinding
import com.androdevdk.camerax_mvvm.test.adapter.PersonAdapter

class ThirdFragment : Fragment() {
    private var _fragmentThirdBinding: FragmentThirdBinding? = null
    private val fragmentThirdBinding get() = _fragmentThirdBinding!!
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        _fragmentThirdBinding = FragmentThirdBinding.inflate(inflater, container, false)
        return fragmentThirdBinding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
    }
    companion object {

    }
}